### PR TITLE
Fixes #6314: s/Feature-Policy/Permissions-Policy/

### LINKED ISSUE #6314: Update reference to Feature-Policy in UA-CH article
In theory this is covered by https://github.com/GoogleChrome/web.dev/issues/3801, but I'm going to send in a PR because we are asking folks to look at https://web.dev/user-agent-client-hints/ now.

### PR DESCRIPTION
**Pull request by miketaylr**
Fixes #6314

Changes proposed in this pull request:

- Update the article to use the new header name
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by miketaylr** [comment]
PTAL @rowan-m!

File: src/site/content/en/blog/user-agent-client-hints/index.md [hunk 1]
```diff
@@ -381,21 +381,21 @@ want to optimize are on `https://downloads.example.com` they **will not**
...
 need to lowercase the hint and remove the `sec-` prefix. For example:
 
 ⬇️ _Response from `example.com`_
 
 ```text
-Accept-CH: Sec-CH-UA-Platform, DPR
-Feature-Policy: ch-ua-platform downloads.example.com;
-                ch-dpr cdn.provider img.example.com
+Accept-CH: Sec-CH-UA-Platform-Version, DPR
+Permissions-Policy: ch-ua-platform-version downloads.example.com;
```
Comment (rowan-m) [inline src/site/content/en/blog/user-agent-client-hints/index.md:391]: I didn't think the format was compatible, is it? As in, I think this needs to be:

```
Permissions-Policy: ch-ua-platform-version=(self "downloads.example.com"),
                    ch-dpr=(self "cdn.provider" "img.example.com")
```

**Review by rowan-m — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/user-agent-client-hints/index.md (hunk 1 above)
Comment (miketaylr) [reply src/site/content/en/blog/user-agent-client-hints/index.md:391]: Wait, you want me to _think_?

**Comment by miketaylr** [comment]
Thanks for catching that! PTAL again, should be fixed.

**Review by rowan-m — APPROVED** [review]
LGTM

**Pull request merged**